crates/biome_css_parser/src/lexer/mod.rs

```diff
@@ -577,10 +578,19 @@ impl<'src> CssLexer<'src> {
         let start = self.text_position();
         while let Some(chr) = self.current_byte() {
             let dispatch = lookup_byte(chr);
-            if matches!(dispatch, PNC) {
-                return CSS_URL_VALUE_RAW_LITERAL;
+            match dispatch {
+                PNC => {
+                    return CSS_URL_VALUE_RAW_LITERAL;
+                }
+                BSL if self.is_valid_escape_at(1) => {
+                    // We can escape any character, so we just skip over the escape sequence
+                    // Even a closing parenthesis (PNC token):
+                    // url(https://example.com/ima\)ge.png);
+                    //                            ^^ escaped closing paren
+                    self.advance(2)
+                }
```

Comment:
Nice! I really wasn't sure how to handle backslashes cleanly here, and this seems really good.